nir/src/main/scala/scala/scalanative/nir/Types.scala

```diff
@@ -176,5 +178,7 @@ object Type {
     case Long   => Sig.Method("longValue", Seq(tpe))
     case Float  => Sig.Method("floatValue", Seq(tpe))
     case Double => Sig.Method("doubleValue", Seq(tpe))
+    case _ =>
+      throw new Exception(s"unknown type ${tpe.show}")
```

Comment:
This is not used, must have been left behind. You may remove `primConvSig` entirely.

Reply:
Deleted, thank you.